Cargo.toml

```diff
@@ -18,6 +18,7 @@ log = "0.3"
 snappy = "0.3"
 ref_slice = "1.0"
 fnv = "1.0"
+openssl = "0.7.10"
```

Comment:
is there a reason to stick exactly to "0.7.10"? i would expect "0.7" would just be fine while automatically receiving bug fixes.